MediaRecorder.html

```diff
@@ -446,8 +446,7 @@
 
           <dt><dfn><code>getSupportedTypes</code></dfn>, static</dt>
           <dd>
-            When <code>getSupportedTypes()</code> method is invoked, the UA MUST
-            queue a task, using the DOM manipulation task source, that runs the
+            When <code>getSupportedTypes()</code> method is invoked, run the
```

Comment:
Can you add "the" here? Or perhaps "this method" which is how isTypeSupported puts it.